common/src/main/java/com/microsoft/identity/common/internal/logging/Logger.java

```diff
@@ -0,0 +1,210 @@
+package com.microsoft.identity.common.internal.logging;
+
+import android.os.Build;
+import android.util.Log;
+
+import com.microsoft.identity.common.adal.internal.util.StringExtensions;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.TimeZone;
+import java.util.concurrent.atomic.AtomicReference;
+
+public final class Logger {
+    private static final Logger INSTANCE = new Logger();
+    private static final String DATE_FORMAT = "yyyy-MM-dd HH:mm:ss";
+
+    // Turn on the VERBOSE level logging by default.
+    private LogLevel mLogLevel  = LogLevel.VERBOSE;
```

Comment:
Is verbose the right default? i think it should be warning.

Reply:
We use the verbose as default in both ADAL and MSAL. I think this is because we want to obtain logs as complete as possible to diagnose when we got an incident.